Anlab.Jobs.MoneyMovement/Program.cs

```diff
@@ -41,70 +39,30 @@ static void Main(string[] args)
 
             IServiceCollection services = new ServiceCollection();
             services.AddDbContext<ApplicationDbContext>(options =>
-                    options.UseSqlite("Data Source=..\\..\\..\\..\\Anlab.Mvc\\bin\\Debug\\netcoreapp1.1\\anlab.db")
+                    options.UseSqlite("Data Source=..\\Anlab.Mvc\\anlab.db")
                 // options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection"))
             );
+            services.AddTransient<ISlothService, SlothService>();           
             Provider = services.BuildServiceProvider();
+            
+            SlothService = Provider.GetService<ISlothService>();           
 
-            Console.WriteLine("About to start");
-            var result = Task.Run(() => ProcessOrders()).Result; //Wasn't able to debug this unless it returned a result...
+            Console.WriteLine("About to start Credit Cards");
+            var financialSettings = new FinancialSettings

```

Comment:
@jSylvestre why did you get away from a separate method that's called from Main() in favor of doing Task.Run directly in main?

Reply:
@srkirkland I just liked it all grouped in the sloth service. If I wasn't going to call the service from the MVC side too, I would have left it here.